webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -198,52 +186,36 @@ class ManageManifestModal extends Component {
 
                     <FormGroup>
                       <Grid>
-                        <h5>{__('Subscription Manifest')}</h5>
+                        <h3>{__('Subscription Manifest')}</h3>
                         <hr />
                         <Row>
-                          <Col sm={4}>
-                            <ControlLabel
-                              className="control-label"
-                              htmlFor="usmaFile"
-                              style={{ paddingTop: '0' }}
-                            >
-                              <OverlayTrigger
-                                overlay={
-                                  <Tooltip id="usma-tooltip">
-                                    {__('Upstream Subscription Management Application')}
-                                  </Tooltip>
-                                }
-                                placement="bottom"
-                                trigger={['hover', 'focus']}
-                                rootClose={false}
-                              >
-                                <div>{__('USMA')}</div>
-                              </OverlayTrigger>
-                            </ControlLabel>
+                          <Col sm={5}>
+                            <strong>{__('Subscription Allocation')}</strong>
                           </Col>
-                          <Col sm={8}>
+                          <Col sm={7}>
                             {getManifestName()}
                           </Col>
                         </Row>
                         <Row>
-                          <Col sm={4}>
-                            <div>{__('Simple Content Access')}</div>
+                          <SimpleContentAccess
+                            enableSimpleContentAccess={enableSimpleContentAccess}
+                            disableSimpleContentAccess={disableSimpleContentAccess}
+                            hideModal={this.hideModal}
+                            isSimpleContentAccessEnabled={simpleContentAccess}
+                            canToggleSimpleContentAccess={!disableSCASwitch}
+                          />
+                          <Col sm={5}>
+                            {canImportManifest &&
+                              <ControlLabel
+                                className="control-label"
+                                style={{ paddingTop: '10 px' }}
+                              >
+                                <div>{__('Upload new manifest')}</div>
```

Comment:
I'm realizing this should probably say "Import new manifest" to match the empty state on the subscriptions page.  Also it's more accurate since it's not really an upload.  (sorry for the one-off)

Reply:
> I'm realizing this should probably say "Import new manifest" to match the empty state on the subscriptions page. Also it's more accurate since it's not really an upload. (sorry for the one-off)

If the manifest is already there we can upload the file and import new manifest but it will not enable the new manifest. So if the manifest is already there should we disable the choose new file option?

Reply:
@piyushtiw No, I think you can import an updated manifest over an existing one as long as the new one is not older than the existing one (right @jturel ?)

Reply:
yep, uploading a new manifest will replace the old one